CPP/UnitTests/SortingAlgrothims_Test.cpp

```diff
@@ -7,9 +7,12 @@
 using namespace DSA::Algrothim::Sort;
 
 static const uint32_t ArrSize = 12;
-static int32_t Arr[]               = {-1,5,2,5,2,0,9,4,7,10,8,-2};
-static int32_t AscendingArr[]      = {-2,-1,0,2,2,4,5,5,7,8,9,10};
-static int32_t DescendingArr[]     = {10,9,8,7,5,5,4,2,2,0,-1,-2};
+static int32_t Arr[]                   = {-1,5,2,5,2,0,9,4,7,10,8,-2};
+static int32_t AscendingArr[]          = {-2,-1,0,2,2,4,5,5,7,8,9,10};
+static int32_t DescendingArr[]         = {10,9,8,7,5,5,4,2,2,0,-1,-2};
+static int32_t Positive_Arr[]          = {10,5,7,2,1,4,6,8,3,0,7,2};
+static int32_t AscendingPositiveArr[]  = {0,1,2,2,3,4,5,6,7,7,8,10};
+static int32_t DescendingPositiveArr[] = {10,8,7,7,6,5,4,3,2,2,1,0};
```

Comment:
You need to think about How to make the algorithm support negative numbers. 
There were a very famous approach to do that :
Suppose we have an array with size 10 then this array can hold positive numbers from 1 : 5 and negative numbers from -1 : -4 where : `arr[0] = 0`  `arr[1] = 1`  .... `arr[6] = -1`